rollouts/manifests/README.md

```diff
@@ -0,0 +1,21 @@
+# manifests
```

Comment:
We can probably either update this to have useful information or just delete it.

Reply:
Ack.